examples/example_bids_datasets.py

```diff
@@ -0,0 +1,187 @@
+#!/group/tuominen/anaconda3/bin/python3
+# Import libraries
+
+import os
+import numpy as np
+import nibabel as nib
+import pandas as pd
+from bids import BIDSLayout
```

Comment:
I suspect you will need to add a requirements.txt to mention pybids need to be installed